DESCRIPTION

```diff
@@ -1,12 +1,34 @@
 Package: miaTime
 Type: Package
-Title: What the Package Does (Title Case)
+Title: Microbiome Time Series Analysis
 Version: 0.1.0
-Author: Who wrote it
-Maintainer: The package maintainer <yourself@somewhere.net>
-Description: More about what it does (maybe more than one line)
-    Use four spaces when indenting paragraphs within the Description.
-License: What license is it under?
+Authors@R:
+     c(person(given = "Leo", family = "Lahti", role = c("aut"),
+              email = "leo.lahti@iki.fi",
+              comment = c(ORCID = "0000-0001-5537-637X")),
+       person(given = "Yagmur", family = "Simsek", role = c("aut", "cre"),
+              email = "yagmur.simsek.98@gmail.com"))
+Description: 
+    Time series analysis of microbiome data sets is evaluated by the functions
+    in `miaTime` package.
+biocViews: Microbiome, Software, Sequencing, Coverage
+License: Artistic-2.0 | file LICENSE
+Depends:
+    SummarizedExperiment
+Imports:
+    methods,
+    TreeSummarizedExperiment
+Suggests: 
+    rmarkdown,
+    knitr,
+    mia,
+    devtools,
+    BiocStyle,
+    testthat (>= 3.0.0)
```

Comment:
There are no specific features in testthat 3.0, which you are using/probably should be using. I would remove the version info here